index.html

```diff
@@ -43,9 +44,12 @@
 		<script type="text/javascript" src="js/app.js"></script>
 		<!-- Load environment variables -->
 		<script src="js/config.js"></script>
+		<!-- Load flavour variables -->
+		<script src="js/flavour.js"></script>
```

Comment:
hmm... It seems there's no `flavour.js` in the `js/` path.

Reply:
It get's generated dynamically for each flavour using:
```
gulp flavour
```

Will use, as you can see, the one found in `/flavour/{$FLAVOUR}/js/flavour.js`